apps/gitness/src/pages/pull-request/stores/pull-request-store.tsx

```diff
@@ -0,0 +1,255 @@
+                  reqCodeOwnerLatestApproval: res.requires_code_owners_approval_latest ?? false,
+                  minReqLatestApproval: res.minimum_required_approvals_count_latest ?? 0,
+                  conflictingFiles: res.conflict_files,
+                  PRStateLoading: false,
+                  commentsLoading: false,
+                  commentsInfoData: {
+                    header: '',
+                    content: undefined,
+                    status: ''
+                  }
+                }
+              } else {
+                draft.prPanelData = {
+                  ruleViolation: false,
+                  ruleViolationArr: undefined,
+                  requiresCommentApproval: res.requires_comment_resolution ?? false,
+                  atLeastOneReviewerRule: res.requires_no_change_requests ?? false,
+                  reqCodeOwnerApproval: res.requires_code_owners_approval ?? false,
+                  minApproval: res.minimum_required_approvals_count ?? 0,
+                  reqCodeOwnerLatestApproval: res.requires_code_owners_approval_latest ?? false,
+                  minReqLatestApproval: res.minimum_required_approvals_count_latest ?? 0,
+                  conflictingFiles: res.conflict_files,
+                  PRStateLoading: false,
+                  commentsLoading: false,
+                  commentsInfoData: {
+                    header: '',
+                    content: undefined,
+                    status: ''
+                  }
+                }
+              }
+            })
+          )
+        })
+        .catch(err => {
+          set(
+            produce(draft => {
+              if (err.status === 422) {
+                draft.prPanelData = {
+                  ruleViolation: true,
+                  ruleViolationArr: err,
+                  requiresCommentApproval: err.requires_comment_resolution ?? false,
+                  atLeastOneReviewerRule: err.requires_no_change_requests ?? false,
+                  reqCodeOwnerApproval: err.requires_code_owners_approval ?? false,
+                  minApproval: err.minimum_required_approvals_count ?? 0,
+                  reqCodeOwnerLatestApproval: err.requires_code_owners_approval_latest ?? false,
+                  minReqLatestApproval: err.minimum_required_approvals_count_latest ?? 0,
+                  conflictingFiles: err.conflict_files,
+                  PRStateLoading: false,
+                  commentsLoading: false,
+                  commentsInfoData: {
+                    header: '',
+                    content: undefined,
+                    status: ''
+                  }
+                }
+              } else if (err.status === 400) {
+                refetchPullReq()
+              } else if (
+                err.message === codeOwnersNotFoundMessage ||
+                err.message === codeOwnersNotFoundMessage2 ||
+                err.message === codeOwnersNotFoundMessage3 ||
```

Comment:
Not an issue, but for repeating things, the `includes` function of `Array` works nicely.
```suggestion
                [codeOwnersNotFoundMessage, codeOwnersNotFoundMessage2, codeOwnersNotFoundMessage3].includes(err.message)
```

And you can even pull that to. variable. Not an issue in the slightest, just an observation.

Reply:
done